stripe/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -61,7 +61,7 @@ internal class DefaultEventReporter internal constructor(
         )
     }
 
-    override fun onSelectPaymentOption(paymentSelection: PaymentSelection) {
+    override fun onSelectPaymentOption(paymentSelection: PaymentSelection?) {
```

Comment:
Why are these nullable? It feels weird to me that the callback is "onSelectPaymentOption" but nothing was selected.

Reply:
This has the useful side-effect of helping track situations where the `paymentSelection` value was lost or in some unexpected state. I think you're right that we don't need to do this everywhere though. Will revisit.